app/src/main/java/com/github/se/travelpouch/model/travels/ListTravelViewModel.kt

```diff
@@ -114,10 +117,16 @@ open class ListTravelViewModel(private val repository: TravelRepository) : ViewM
 
   /** Gets all Travel documents. */
   fun getTravels() {
-
+    _isLoading.value = true
```

Comment:
Worry not, the icon will disappear in the situation you describe. That is because _isLoading is set to false whether it succeeds or not. The only way for it to keep spinning is if neither callback is called. This maybe could happen if there's a timeout.